apps/trading/components/navbar/navbar.tsx

```diff
@@ -75,12 +79,30 @@ export const Navbar = ({
           </NavigationLink>
         </NavigationItem>
         <NavigationItem>
-          <ExternalLink href={tokenLink(TOKEN_GOVERNANCE)}>
-            <span className="flex items-center gap-2">
-              <span>{t('Governance')}</span>{' '}
-              <Icon name="arrow-top-right" size={3} />
-            </span>
-          </ExternalLink>
+          <NavExternalLink href={tokenLink(TOKEN_GOVERNANCE)}>
+            {t('Governance')}
+          </NavExternalLink>
+        </NavigationItem>
+        <NavigationItem>
+          <NavigationTrigger>{t('Resources')}</NavigationTrigger>
```

Comment:
question (non-blocking): if no VEGA_DOCS_URL nor GITHUB_FEEDBACK are given, Resources dropdown will be empty?

Reply:
Done